jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -266,6 +266,10 @@ static Unsafe getUnsafe() {
 	private Class<?> nestHost;
 /*[ENDIF] Java11*/
 	
+/*[IF Java15]*/
+	private transient Object classData;
```

Comment:
Is this field visible to Reflection in the RI?

Reply:
The `Class.classData` field is listed in [Reflection.fieldFilterMap](https://github.com/ibmruntimes/openj9-openjdk-jdk15/blob/88d915c8537bf7e6a9828c0523cb7478cf20ed10/src/java.base/share/classes/jdk/internal/reflect/Reflection.java#L55). It gets filtered out from public-view. I don't think it is visible to Reflection in the RI. Assuming, I have correctly read the code.